day2_configuration.example.yaml

```diff
@@ -21,3 +21,8 @@ configurators:
         cluster_domain: <ocp.customer.domain.com>
         ca_pem_file_path: </path/to/ca.pem>
         ca_key_file_path: </path/to/ca.key>
+
+    nodes:
+        cluster_domain: <ocp.customer.domain.com>
```

Comment:
this is the same one as in `ingress`, if so should be defined once

Reply:
what if we don't use ingress in the configuration file?
or else we are expected to run all configurators?

Reply:
should be top level config

Reply:
done